.env.appStore.example

```diff
@@ -125,4 +125,9 @@ SALESFORCE_CONSUMER_SECRET=""
 ZOHOCRM_CLIENT_ID=""
 ZOHOCRM_CLIENT_SECRET=""
 
+#  - ZOHOCALENDAR
+# Used for the Zoho Calendar integration
+ZOHOCALENDAR_CLIENT_ID=""
+ZOHOCALENDAR_CLIENT_SECRET=""
```

Comment:
Let's remove this. This isn't needed now. `zod.ts` has this added already and thus they would be available to edit in settings/admin/apps.